shell/artifacts/Profile/Profile.recipes

```diff
@@ -10,7 +10,7 @@ import 'FriendsPicker.manifest'
 import 'FavoriteFoodPicker.manifest'
 import '../Places/AddressForm.manifest'
 
-recipe
+recipe BasicUserProfile
```

Comment:
Intentional here and below?

Reply:
Naming recipes makes them easier to track while debugging. I didn't know this was a thing until recently.